spec/dummy/app/avo/resources/person.rb

```diff
@@ -10,7 +10,7 @@ class Avo::Resources::Person < Avo::BaseResource
 
   def fields
     field :name, as: :text, link_to_record: true, sortable: true, stacked: true
-    field :type, as: ::Pluggy::Fields::RadioField, name: "Type", options: {Spouse: "Spouse", Sibling: "Sibling"}, include_blank: true, filterable: true
+    field :type, as: :pluggy_radio, name: "Type", options: {Spouse: "Spouse", Sibling: "Sibling"}, include_blank: true, filterable: true
```

Comment:
Is this change really necessary? I think it's a good test for declaring fields using the constant.

Reply:
Will change to use the constant then! It's not necessary, just thought would be a better practice.